homeassistant/components/econet/water_heater.py

```diff
@@ -158,22 +138,18 @@ def temperature_unit(self):
     def device_state_attributes(self):
         """Return the optional device state attributes."""
```

Comment:
Users that have been testing out my version of these changes might be expecting 

ATTR_WIFI_SIGNAL = "wifi_signal_strength"
ATTR_ALERT_COUNT = "alert_count"

To be present in the state attributes, you may want to add those in.

If you are feeling crazy, it would be a good idea to create a base EcoNet entity which has all of the common stuff in it so the next MR for HVAC is easier. But not a requirement of course.